packages/react/src/primitives/Alert/AlertIcon.tsx

```diff
@@ -22,26 +23,37 @@ interface AlertIconProps {
 export const AlertIcon = ({
   variation,
   ariaHidden,
+  ...rest
 }: AlertIconProps): JSX.Element | null => {
   const icons = useIcons('alert');
   let icon;
   switch (variation) {
     case 'info':
-      icon = icons?.info ?? <IconInfo aria-hidden={ariaHidden} />;
+      icon = icons?.info ?? <IconInfo aria-hidden={ariaHidden} {...rest} />;
       break;
     case 'error':
-      icon = icons?.error ?? <IconError aria-hidden={ariaHidden} />;
+      icon = icons?.error ?? <IconError aria-hidden={ariaHidden} {...rest} />;
       break;
     case 'warning':
-      icon = icons?.warning ?? <IconWarning aria-hidden={ariaHidden} />;
+      icon = icons?.warning ?? (
+        <IconWarning aria-hidden={ariaHidden} {...rest} />
+      );
       break;
     case 'success':
-      icon = icons?.success ?? <IconCheckCircle aria-hidden={ariaHidden} />;
+      icon = icons?.success ?? (
+        <IconCheckCircle aria-hidden={ariaHidden} {...rest} />
+      );
       break;
   }
 
   return icon ? (
-    <span className={ComponentClassName.AlertIcon}>{icon}</span>
+    <span
+      className={ComponentClassName.AlertIcon}
+      aria-hidden={ariaHidden}
+      {...rest}
```

Comment:
Can we double check the output of this HTML? (a nested img role inside of an img role, both with an aria-label) Maybe how we're passing ...rest props down or something?

<img width="557" alt="Screenshot 2024-01-05 at 11 09 50 AM" src="https://github.com/aws-amplify/amplify-ui/assets/376920/1b81789c-1382-4e52-affe-aa67284f4055">

It reads fine in VoiceOver, but it's the sort of weird HTML that might cause issues for screen readers.

Reply:
Should be fixed updated screenshot